DaandB/GDEV_Huiswerk1.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class playerController : MonoBehaviour {
+
+    public float speed { get; set; } = 10f;
```

Comment:
Auto property initializer kan pas gebruikt worden in C# versie 6 of hoger en kan dus problemen geven voor mensen met een oudere versie. Je zou een private aan kunnen maken met een standaard waarde en die gebruiken voor je getter/setter

```
[SerializeField] private float _speed= 10f;
public float Speed {
    get { return _speed; }
    set { Speed = value; }
}
```